README.md

```diff
@@ -12,8 +12,8 @@ This is based on https://github.com/ryanlovett/nbnovnc.
 
 When this extension is launched it will run a Linux desktop on the Jupyter single-user server, and proxy it to your browser using VNC via Jupyter.
 
-If a `vncserver` executable is found in `PATH` it will be used, otherwise a bundled TightVNC server is run.
-You can use this to install vncserver with support for other features, for example the [`Dockerfile`](./Dockerfile) in this repository installs TurboVNC for improved OpenGL support.
+This extension requires a VNC server to be installed on the system, along with a desktop or equivalent.
+For example, see the [`Dockerfile`](./Dockerfile) in this repository which installs TigerVNC and XFCE4.
```

Comment:
```suggestion
For example, see the [`Dockerfile`](./Dockerfile) in this repository which installs [TigerVNC](https://tigervnc.org/) and XFCE4.
```

Reply:
I think this was resolved by [94cf5b6](https://github.com/jupyterhub/jupyter-remote-desktop-proxy/pull/84/commits/94cf5b636de529f376e36f473908474d54f8bc05)